backend/migration.js

```diff
@@ -0,0 +1,229 @@
+const path = require('path');
+const Umzug = require('umzug');
+const _ = require('lodash');
+
+function onMigrationEnd(exitCode) {
+    // eslint-disable-next-line no-process-exit
+    process.exit(exitCode);
+}
+
+function getCurrent(executed) {
+    return _.chain(executed).last().get('file').value() || '<NO_MIGRATIONS>';
+}
+
+/**
+ * Runs migration script
+ *
+ * @param {Object} loggerFactory initialized logger factory
+ * @param {Object} dbModule DB module, an object containing `init` function and `db` object
```

Comment:
Here we can also make reference to `db.js`. Probably you can make use of [@link](https://jsdoc.app/tags-inline-link.html) or [@see](https://jsdoc.app/tags-see.html) .

Reply:
Added